adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -497,7 +498,8 @@ func createRoute(params *routeCreateParams) *routev3.Route {
 					},
 					Substitution: endpointBasepath,
 				},
-				UpgradeConfigs: getUpgradeConfig(apiType),
+				UpgradeConfigs:    getUpgradeConfig(apiType),
+				MaxStreamDuration: getMaxStreamDuration(apiType),
```

Comment:
with envoy 1.18, we have to configure routes for `MaxStreamDuration`. getMaxStreamDuration returns max stream duration for websocket and for http it returns nil (no change for http) https://www.envoyproxy.io/docs/envoy/v1.18.2/version_history/v1.18.0 cc @VirajSalaka